cinema_service/urls.py

```diff
@@ -6,4 +8,4 @@
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("api/user/", include("user.urls", namespace="user")),
     path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
The debug toolbar is included in the URL patterns. Make sure to conditionally include this only in a development environment to avoid exposing debug information in production.